pkgdb/src/flox-flake.cc

```diff
@@ -37,22 +37,84 @@
 namespace flox {
 
 /* -------------------------------------------------------------------------- */
+void
+ensureFlakeIsDownloaded( std::function<void()> && lambda )
+{
+  pid_t pid = fork();
+  if ( pid == -1 )
+    {
+      // WML - TODO - better error handling here!
+      errorLog( nix::fmt(
+        "ensureFlakeIsDownloaded: faild to fork for flake downlod!" ) );
+      exit( -1 );
+    }
+  if ( 0 < pid )
```

Comment:
Nit: I think it's more common to write this as `pid > 0`, I actually read this incorrectly the first time even though it's pretty simple.

Reply:
The two forms are equivalent so either is acceptable.
My personal preference is _less than_ but I can't actually remember why ( superstition probably ).

For our purposes I don't think `x > y` vs `y < x` is a style rule we ought to enforce.